src/AtelierTomato.Markov.Service.Discord/DiscordSentenceBuilder.cs

```diff
@@ -0,0 +1,23 @@
+﻿using AtelierTomato.Markov.Model;
+using AtelierTomato.Markov.Model.ObjectOID;
+using Discord;
+
+namespace AtelierTomato.Markov.Service.Discord
+{
+	public class DiscordSentenceBuilder
+	{
+		public static async Task<IEnumerable<Sentence>> Build(IGuild? guild, IChannel channel, ulong messageID, ulong userID, DateTimeOffset date, IEnumerable<string> sentenceTexts, string instance = "discord.com")
+		{
+			DiscordObjectOID OID = await DiscordObjectOIDBuilder.Build(guild, channel, messageID, instance);
+			AuthorOID author = new(ServiceType.Discord, instance, userID.ToString());
+			return sentenceTexts.Select(text =>
+				new Sentence(
+					OID.IncrementSentence(),

```

Comment:
yeah this will break because all sentences will have the same OID. mutation is bad.

Reply:
a well written unit test could catch this.